torchao/quantization/README.md

```diff
@@ -216,7 +216,12 @@ This technique works best when the torch._inductor.config.use_mixed_mm option is
 ```python
 # for torch 2.4+
 from torchao.quantization import quantize_, int4_weight_only
-quantize_(model, int4_weight_only())
+group_size = 32
+
+# you can enable [hqq](https://github.com/mobiusml/hqq/tree/master) quantization which is expected to improves accuracy through
+# use_hqq flag for `int4_weight_only` quantization
+use_hqq = False
+quantize_(model, int4_weight_only(group_size=group_size, use_hqq=use_hqq))
```

Comment:
since this is user facing, should the current bool `use_hqq` instead be an enum, something like `SingleWeightQuantizationAlgorithm.HQQ`?

Reply:
yeah this is not ideal, I'm planning to just have a separate hqq config and remove the flag

Reply:
@vkuzo I'm integrating hqq into `uintx_weight_only` API now, and I'm keeping the boolean flag for now to keep it simpler, we can make this an enum if there are more algorithms in the future I think, please let me know if that sounds OK

Reply:
if you're ok with potentially changing it later, sgtm